Automattic/wp-calypso
# Gallery Shortcode: send render REST request with a simple inline hook

Removes the entire `state/shortcode` Redux state and replaces it with an inline hook that calls the `wpcom.undocumented().site().shortcodes()` request.

There is only one usage of this request in `Shortcode` component and it doesn't need any caching or sharing.

**How to test:**
1. Open a post in the Gutenframe editor
2. Create a Classic block
3. Click "Add Media" in the Classic block toolbar
4. Media Modal opens. Select several images and click "Continue"

Now you see a Gallery Preview in the modal:

<img width="1070" alt="Screenshot 2021-02-16 at 12 49 59" src="https://user-images.githubusercontent.com/664258/108062357-fc2add80-7059-11eb-9084-c72d3005bc28.png">

As you modify the gallery options on the right, the preview (rendered as iframe with shortcode preview) should change accordingly.
